namespaces/http.yml

```diff
@@ -0,0 +1,59 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+namespace:
+  name: http
```

Comment:
ok - then it sounds like it is generic enough to be http - would like @portante and @t0ffel to take a look too - they probably have a lot of experience with http logs in their performance lab